http/http-api/src/main/java/com/okta/commons/http/MimeType.java

```diff
@@ -536,7 +538,7 @@ public static MimeType valueOf(String value) {
     }
 
 
-    public static class SpecificityComparator<T extends MimeType> implements Comparator<T> {
+    public static class SpecificityComparator<T extends MimeType> implements Comparator<T>, Serializable {
```

Comment:
Fixes below Findbugs error:

`com.okta.commons.http.MimeType$SpecificityComparator implements Comparator but not Serializable [com.okta.commons.http.MimeType$SpecificityComparator] At MimeType.java:[lines 539-571] SE_COMPARATOR_SHOULD_BE_SERIALIZABLE`